app/services/sufia/workflow/complete_notification.rb

```diff
@@ -0,0 +1,25 @@
+module Sufia
+  module Workflow
+    class CompleteNotification
+      def self.send_notification(entity:,
```

Comment:
Seems like in these two classes there is a bit of repeated code. Do we maybe want to make an object that builds a notification? @jcoyne @mjgiarlo Just a thought really.